.env.appStore.example

```diff
@@ -137,3 +138,7 @@ REVERT_PUBLIC_TOKEN=
 # NOTE: If you're self hosting Revert, update this URL to point to your own instance.
 REVERT_API_URL=https://api.revert.dev/
 # *********************************************************************************************************
+
+#   - Huddle01
+# Used for the huddle01 integration
+HUDDLE01_API_TOKEN=
```

Comment:
same here

Reply:
I suggest that you leave one extra line. It currently ends at 144, line should end at 145. Just press Enter